src/button.js

```diff
@@ -38,11 +59,13 @@ class Button extends Component {
     },
 
     Enter(event) {
-      event.preventDefault()
-      if (this.autocomplete.state.isOpen) {
-        this.autocomplete.selectHighlightedItem()
-      } else {
-        this.autocomplete.openMenu()
+      if (this.props.action !== 'clearSelection') {
```

Comment:
@kentcdodds any ideas how to handle this condition if we aren't using an action prop?